src/key_variants/double_key.rs

```diff
@@ -78,19 +90,17 @@ impl Serializable<64> for PublicKeyPair {
 #[derive(Clone, Copy, Debug)]
 #[cfg_attr(feature = "canon", derive(Canon))]
 pub struct Proof {
-    U: JubJubScalar,
-    R: PublicKeyPair,
+    u: JubJubScalar,
```

Comment:
Good catch, I ported the original code without noticing the wrong naming convention here.